app/views/markets/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<section>
+  <%= link_to "Create a New Market", new_market_path %>
+</section>
+
+<article class="container">
+  <% @markets.each do |market| %>
+    <section class="market-card">
+      <span class="m-card"><%= link_to market.name, market_path(market.id) %></span>
+      <span class="m-card"><%= market.address %></span>
```

Comment:
Instead of giving all of these span elements the "m-card" class, in your css file, you could select all children of the class "market-card." Your way is not wrong, but this is another way that would require less typing. 🌸